### DISCUSSION TITLE
How do Remap "Ctrl + a" to select all lines of code in a file?

### DISCUSSION THREAD

**Discussion by jarmos-san** [VSCodeVim/Vim#7715] [Q&A]
In Neo(vim) I've configured the editor to select all the lines of code in the current buffer by pressing "_Ctrl + a_". The keys are remapped to `ggVG` which works flawlessly natively but not when using the plugin in VSCode.

Any idea how can I configure the plugin to achieve what I want? Or am I missing something out?

**Comment by J-Fields** [comment]
Using a [.vimrc](https://github.com/VSCodeVim/Vim#vimrc-support), this works just fine:

`nnoremap <C-a> ggVG`

**Reply by jarmos-san** [reply]
Cool...but I was wondering if there was a way to do the same through `settings.json`?
